hometask1/foo.rb

```diff
@@ -0,0 +1,46 @@
+class Foo
+  @@counter = 0
+  @instance_counter = 0
+
+  def initialize
+    self.class.counter += 1
+    @instance_counter = Foo.counter
+  end
+
+  def self.counter=value
+    @@counter=value
+  end
+
+  def self.counter
+    @@counter
+  end
+
+  def my_instance
+  if @instance_counter%2==0
```

Comment:
Take a look on [#even](https://ruby-doc.org/core-2.3.1/Integer.html#method-i-even-3F) method.
And _even_, not _eval_.